css/production/mail.css

```diff
@@ -1,56 +1,99 @@
 #page-wrapper {
-  max-width: 100%;
-  padding: 5px; }
-  #page-wrapper .col-lg-12 {
+    max-width: 100%;
+    padding: 5px;
+}
+
+#page-wrapper .col-lg-12 {
     padding: 0 !important;
-    max-width: 100%; }
-  #page-wrapper .row {
-    margin: 0; }
-  #page-wrapper #mailTable th {
+    max-width: 100%;
+}
+
+#page-wrapper .row {
+    margin: 0;
+}
+
+#page-wrapper #mailTable th {
     max-width: 100px;
-    min-width: 100px; }
-    #page-wrapper #mailTable th .subject {
-      max-width: 80%; }
-  #page-wrapper #mailTable td {
-    max-height: 500px; }
-    #page-wrapper #mailTable td iframe {
-      width: 100%;
-      height: 100%; }
-  #page-wrapper #mailTable:hover {
-    cursor: default !important; }
+    min-width: 100px;
+}
+
+#page-wrapper #mailTable th .subject {
+    max-width: 80%;
+}
+
+#page-wrapper #mailTable td {
+    max-height: 500px;
+}
+
```

Comment:
Inte hemma så får ta och fixa det imorgon isf!

On Monday, 16 May 2016, Emil Hammarström <notifications@github.com> wrote:

> In css/production/mail.css
> <https://github.com/initiumSrc/QuickFeed-app/pull/13#discussion_r63380923>
> :
>
> > -    #page-wrapper #mailTable td iframe {
> > -      width: 100%;
> > -      height: 100%; }
> > -  #page-wrapper #mailTable:hover {
> > -    cursor: default !important; }
> > +    min-width: 100px;
> > +}
> > +
> > +#page-wrapper #mailTable th .subject {
> > +    max-width: 80%;
> > +}
> > +
> > +#page-wrapper #mailTable td {
> > +    max-height: 500px;
> > +}
> > +
>
> Släng in det i scssen (annars skrivs det över vid gulp test eller gulp
> build), och det blir lite mer lättläsligt när det är så nästlad css.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/initiumSrc/QuickFeed-app/pull/13/files/fd3a5091e86c76e0f900b96e1e51bc2c65a44c4e#r63380923>
>